EventGenerator/src/MuStopProductsGun_module.cc

```diff
@@ -0,0 +1,156 @@
+// Andy Edmonds, 2020
+// based on StoppedParticleReactionGun by Andrei Gaponenko, 2013
+
+#include <iostream>
+#include <string>
+#include <cmath>
+#include <memory>
+#include <algorithm>
+
+#include "cetlib_except/exception.h"
+
+#include "CLHEP/Vector/ThreeVector.h"
+#include "CLHEP/Vector/LorentzVector.h"
+#include "CLHEP/Random/RandomEngine.h"
+#include "CLHEP/Random/RandGeneral.h"
+#include "CLHEP/Random/RandPoissonQ.h"
+#include "CLHEP/Units/PhysicalConstants.h"
+
+#include "art/Framework/Core/EDProducer.h"
+#include "art/Framework/Core/ModuleMacros.h"
+#include "art/Framework/Principal/Event.h"
+#include "art/Framework/Principal/Run.h"
+#include "art/Framework/Principal/Handle.h"
+#include "art/Framework/Services/Registry/ServiceHandle.h"
+#include "art/Utilities/make_tool.h"
+
+#include "fhiclcpp/ParameterSet.h"
+#include "fhiclcpp/types/DelegatedParameter.h"
+#include "fhiclcpp/types/OptionalAtom.h"
+
+#include "ConfigTools/inc/ConfigFileLookupPolicy.hh"
+#include "SeedService/inc/SeedService.hh"
+#include "GlobalConstantsService/inc/GlobalConstantsHandle.hh"
+#include "GlobalConstantsService/inc/ParticleDataTable.hh"
+#include "GlobalConstantsService/inc/PhysicsParams.hh"
+#include "DataProducts/inc/PDGCode.hh"
+#include "MCDataProducts/inc/GenParticle.hh"
+#include "MCDataProducts/inc/GenParticleCollection.hh"
+#include "Mu2eUtilities/inc/RandomUnitSphere.hh"
+#include "Mu2eUtilities/inc/CzarneckiSpectrum.hh"
+#include "Mu2eUtilities/inc/ConversionSpectrum.hh"
+#include "Mu2eUtilities/inc/SimpleSpectrum.hh"
+#include "Mu2eUtilities/inc/EjectedProtonSpectrum.hh"
+#include "Mu2eUtilities/inc/BinnedSpectrum.hh"
+#include "Mu2eUtilities/inc/Table.hh"
```

Comment:
Please remove the leftover includes.  Also RandPoissonQ.h

Reply:
Done in commit 27d481d68